src/lib/components/works_for_you/notification.tsx

```diff
@@ -0,0 +1,102 @@
+import * as React from "react"
+import { StyleSheet, View, ViewProperties } from "react-native"
+import * as Relay from "react-relay"
+
+import ArtworksGrid from "../artwork_grids/generic_grid"
+import Headline from "../text/headline"
+import SerifText from "../text/serif"
+
+import colors from "../../../data/colors"
+
+interface Props extends ViewProperties {
```

Comment:
Generally the idea is to also extend the `RelayProps` interface that’s at the bottom of this file, rather than writing those props fetched by Relay by hand.